contracts/Bridge.sol

```diff
@@ -1,4 +1,5 @@
-pragma solidity 0.5.10;
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
why was this line added?

Reply:
I was receiving the following warning if i do not add this line to the contracts:
SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.

Adding this line caused the warning to go away, it seems like the following warning comes up in solidity 6+